src/renderers/shared/fiber/ReactFiberErrorLogger.js

```diff
@@ -25,16 +25,25 @@ function logCapturedError(capturedError : CapturedError) : void {
       willRetry,
     } = capturedError;
 
-    const errorSummary = error.message
-      ? `Error: ${error.message}\n\n`
+    const {
+      message,
+      name,
+      stack,
+    } = error;
+
+    const errorSummary = message
+      ? `${name}: ${message}\n\n`
       : '';
 
     const componentNameMessage = componentName
       ? `React caught an error thrown by ${componentName}.`
       : 'React caught an error thrown by one of your components.';
 
-    const callStack = error.stack;
-    const trimmedCallStack = callStack.substring(callStack.indexOf('\n'));
+    // Error stack varies by browser; normalize it for our logging.
+    let formattedCallStack = stack.indexOf(message) >= 0
```

Comment:
Agreed. Hadn't considered the multiline case. Shouldn't code before ☕️ 😁

I think this version (just pushed) is more robust. Tested various browsers with single-line message, multi-line message, and no message.